polls/migrations/0006_annotatorprofile.py

```diff
@@ -0,0 +1,38 @@
+# Generated by Django 4.1.1 on 2023-02-24 12:05
+
+import django.db.models.deletion
+from django.conf import settings
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("polls", "0005_alter_batch_created_at"),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name="AnnotatorProfile",
+            fields=[
+                (
+                    "id",
+                    models.BigAutoField(
+                        auto_created=True,
+                        primary_key=True,
+                        serialize=False,
+                        verbose_name="ID",
+                    ),
+                ),
+                ("email", models.EmailField(blank=True, max_length=254)),
+                ("hourly_rate", models.FloatField(blank=True, default=None)),
+                (
+                    "anotator",
+                    models.ForeignKey(
+                        on_delete=django.db.models.deletion.CASCADE,
+                        related_name="anotator",
+                        to=settings.AUTH_USER_MODEL,
```

Comment:
Perhaps this is my lack of Django but why is this a separate table from "Annotator"? I mean I guess this is fine

Reply:
(Or user model. But maybe it makes sense because some users are admins and some are annotators. In that case shouldn't the fk be called user?)

Reply:
Actually, that is somehow how django works:
When you define a foreign key relationship between two models in Django, a new table is created to represent the relationship, this table is used to store the primary key representing the relationship and it ease querying data from either of the table, and this table is also used to maintain the relationship between the two models.

Reply:
Also, the fk related name has been changed to user